scripts/build-node-scripts.sh

```diff
@@ -4,13 +4,18 @@ BABEL=$PWD/node_modules/.bin/babel
 
 mkdir -p $OUTDIR/server/js &&
 mkdir -p $OUTDIR/shared/js &&
-mkdir -p $OUTDIR/shared/partials &&
```

Comment:
Note: I moved this to the `build:node:views` task, as I think it makes more sense for it to be there.